src/pages/ExamsPage/data/reducer.js

```diff
@@ -36,6 +36,27 @@ const slice = createSlice({
       ...state,
       attemptsList: state.attemptsList.filter(attempt => attempt.attempt_id !== attemptId.payload),
     }),
+    modifyExamAttempt: (state, attemptId, action, attemptIndex) => ({
+      ...state,
+      // TODO: Make the status of the attempt modified change in the UI
+      attemptsList: state.attemptsList.map((attempt, index) => {
+        // Set the status of the modified attempt to verified or rejected
+        debugger;
+        if (index === attemptIndex) {
+          return {
+            ...attempt,
+            status: (() => {
+              if (action === 'verify') {
```

Comment:
the fact this falls through to rejected on any string that isn't 'verify' might be bug prone. Id suggest making this mapping more specific. Maybe a switch statement would be cleaner here?